### DISCUSSION TITLE
Installing a pre-release version without building from scratch?

### DISCUSSION THREAD

**Discussion by BatmanAoD** [carapace-sh/carapace-bin#2554] [Q&A] [answered]
Is it possible to install a pre-release version without building from scratch? I'd like to get the [merged `jj` update](https://github.com/carapace-sh/carapace-bin/pull/2544), but it looks like releases happen on a monthly cadence so these won't be available in the primary release for another few weeks.

It would be great if there were a "nightly channel" to be able to install from.

**Comment by rsteube** [comment, marked as answer]
Yeah, I've been meaning to add a nightly build for some time now.
Where to put It and how to handle the update mechanism is something I need  to give some thinking though.

**Reply by rsteube** [reply]
https://carapace-sh.github.io/carapace-bin/install/selfupdate.html
